compiler/fixture_dirs.txt

```diff
@@ -1,95 +1,95 @@
-oss/crates/dependency-analyzer/tests/ast
```

Comment:
Removing these prefixes is included in D42421902. I'll rebase once that lands and this change should go away.